### ISSUE TITLE
是否存在不支持的层导致的精度损失

### ISSUE THREAD

**Issue by GeorgeBohw** [espressif/esp-dl#314]
### Checklist

- [x] Checked the issue tracker for similar issues to ensure this is not a duplicate.
- [x] Provided a clear description of your suggestion.
- [x] Included any relevant context or examples.

### Issue or Suggestion Description

class net(nn.Module):
    def __init__(self, seq_len=180, dropout=0.4):
        super().__init__()
        self.seq_len = seq_len

        self.spatial = nn.Sequential(
            nn.Conv2d(2, 32, 3, padding=1),
            nn.BatchNorm2d(32), nn.ReLU(),
            nn.Conv2d(32, 64, 3, padding=1),
            nn.BatchNorm2d(64), nn.ReLU(),
        )
        self.spatial_out_dim = 64 * 4 * 4
        self.spatial_attn = SpatialAttention()
        self.spatial_down = nn.Conv2d(64, 64, kernel_size=3, stride=2, padding=1)

        self.temp_short = nn.Sequential(
            nn.Conv1d(self.spatial_out_dim, 128, kernel_size=3, padding=0, dilation=1),
            nn.BatchNorm1d(128), nn.ReLU()
        )

        self.temp_mid = nn.Sequential(
            nn.Conv1d(self.spatial_out_dim, 128, kernel_size=3, padding=0, dilation=2),
            nn.BatchNorm1d(128), nn.ReLU()
        )

        self.temp_long = nn.Sequential(
            nn.Conv1d(self.spatial_out_dim, 128, kernel_size=3, padding=0, dilation=4),
            nn.BatchNorm1d(128), nn.ReLU()
        )

        self.temporal_attn = TemporalAttention(384)

        self.classifier = nn.Sequential(
            nn.Linear(384, 128),
            nn.ReLU(),
            nn.Dropout(dropout),
            nn.Linear(128, 64),
            nn.ReLU(),
            nn.Linear(64, 1)
        )

        init_weights(self)

    def forward(self, x):
        # if x.dim() == 5 and x.shape[2] == 1:
        #     x = x.permute(0, 2, 1, 3, 4)

        # if x.shape[2] == 180:
        #     x = x[:, :, ::6, :, :]

        # B, C, S, H, W = x.shape

        c0 = x
        first_frame = x[:, :, 0:1, :, :]
        c1 = x - first_frame

        x_multi = torch.cat([c0, c1], dim=1)

        x_s = x_multi.permute(0, 2, 1, 3, 4).contiguous().view(30, 2, 8, 8)
        x_s = self.spatial(x_s)
        x_s = self.spatial_attn(x_s)
        x_s = self.spatial_down(x_s)
        x_s = x_s.view(1, 30, 1024).permute(0, 2, 1)

        x_s_short = F.pad(x_s, (2, 0), mode='replicate')
        x_short = self.temp_short(x_s_short)

        x_s_mid = F.pad(x_s, (4, 0), mode='replicate')
        x_mid = self.temp_mid(x_s_mid)

        x_s_long = F.pad(x_s, (8, 0), mode='replicate')
        x_long = self.temp_long(x_s_long)

        x_t = torch.cat([x_short, x_mid, x_long], dim=1)

        # pos = torch.linspace(0, 1, 30, device=x_t.device)
        # pos = pos.unsqueeze(0).unsqueeze(0)
        # x_t = x_t + pos

        cnn_feat = self.temporal_attn(x_t)

        logits = self.classifier(cnn_feat).squeeze(-1)

        # result = torch.sigmoid(logits)

        return logits

该模型转换为.espdl，发现端侧的输出和pytorch的输出差别较大，请帮忙确认网络中是否有不支持的层，或者哪一层导致了输出发生了严重的偏差。谢谢。

**Comment by GeorgeBohw** [comment]
@sun-xiangyu 谢谢

**Comment by sun-xiangyu** [comment]
不存在这样的情况，如果有不支持的算子，会在模型初始化的时候报错，无法运行
